docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/index.md

```diff
@@ -61,175 +61,181 @@ In this section, we'll introduce the following concepts:
+<div className="box smallbox12 card">
   <div className="container">
   <a href="/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/carbon-black-inventory-source"><img src={useBaseUrl('img/integrations/security-threat-detection/vmcarecb.png')} alt="thumbnail icon" width="50"/><h4>Carbon Black Inventory</h4></a>
   <p>Provides a secure endpoint to receive data from the CB Devices API.</p>
   </div>
 </div>
-<div className="box smallbox12 card">
+<div className="box smallbox13 card">
   <div className="container">
   <a href="/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/cato-networks-source"><img src={useBaseUrl('img/send-data/cato-logo.png')} alt="icon" width="90"/><h4>Cato Networks</h4></a>
   <p>Cato Networks Source helps to retrieve Cato audit and security logs into the Sumo Logic environment.</p>
   </div>
 </div>
-<div className="box smallbox13 card">
+<div className="box smallbox14 card">
   <div className="container">
   <a href="/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/cisco-amp-source"><img src={useBaseUrl('img/send-data/cisco-amp.png')} alt="icon" width="90"/><h4>Cisco AMP</h4></a>
   <p>Provides a secure endpoint to receive data from the Cisco Amp System Log API.</p>
   </div>
 </div>
-<div className="box smallbox14 card">
+<div className="box smallbox15 card">
   <div className="container">
   <a href="/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/cisco-meraki-source"><img src={useBaseUrl('img/send-data/cisco-meraki-sumo-source-select.png')} alt="icon" width="90"/><h4>Cisco Meraki</h4></a>
   <p>Provides a secure endpoint to receive data from the Cisco Meraki API.</p>
   </div>
 </div>
-<div className="box smallbox15 card">
+<div className="box smallbox16 card">
   <div className="container">
   <a href="/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/citrix-cloud-source"><img src={useBaseUrl('img/send-data/citrix-cloud-icon.png')} alt="icon" width="100"/><h4>Citrix Cloud Source</h4></a>
   <p>Provides a secure endpoint to receive System Log data from the Citrix Cloud System Log API.</p>
   </div>
 </div>
-<div className="box smallbox16 card">
+<div className="box smallbox17 card">
   <div className="container">
   <a href="/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/cloud-to-cloud-source-versions"><img src={useBaseUrl('img/icons/business/SaaS.png')} alt="icon" width="50"/><h4>Cloud-to-Cloud Versions</h4></a>
   <p>Sources in the Cloud-to-Cloud Integration Framework need updates over time to maintain data collection.</p>
   </div>
 </div>
-<div className="box smallbox17 card">
+<div className="box smallbox18 card">
   <div className="container">
   <a href="/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/crowdstrike-source"><img src={useBaseUrl('img/integrations/security-threat-detection/crowdstrike.png')} alt="thumbnail icon" width="65"/><h4>CrowdStrike</h4></a>
```

Comment:
```suggestion
```

Reply:
This should be removed. It appears in error in the Code42 card.

Reply:
Thanks for pointing out this John. I have updated this.